packages/lib-classifier/src/components/Classifier/components/Layout/components/DefaultLayout/DefaultLayout.js

```diff
@@ -9,26 +10,14 @@ import QuickTalk from '@components/Classifier/components/QuickTalk'
 import SubjectViewer from '@components/Classifier/components/SubjectViewer'
 import TaskArea from '@components/Classifier/components/TaskArea'
 
-const ContainerGrid = styled.div`
-  display: grid;
-  grid-gap: 30px;
-  grid-template-areas: "viewer task";
-  grid-template-columns: auto 25.333rem;
+const ContainerGrid = styled(Grid)`
   position: relative;
 
-  @media screen and (max-width: 1160px) {
+  // proportional 9/5 subject/task sizing up to a maximum task width of 25.333rem.
+  @media screen and (min-width: 769px) and (max-width: 70.666rem) {
```

Comment:
Another way of thinking about this:
Why not put `columns: [45.333fr, 25.333fr]` in the `horizontalLayout` props and make this media query only for screens larger than 70.666rem? Am I right in understanding that screens larger than 70.666rem is the case when you want the grid columns to equal `[auto, 25.333fr]`? In that case, the grommet Grid would only by overwritten by styled-components in that specific widescreen case.

Reply:
This query fixes #3336 and gives 9/5 proportional subject/task sizing for intermediate screen sizes or larger base font sizes eg. when Larger Text is enabled in MacOS.

Reply:
`horizontalLayout` doesn’t respond to text size settings. If I remember correctly, Grommet’s responsive context is based on fixed pixel sizes, rather than `rem`.

Reply:
Ah, I think the units were confusing me. For instance, the 25.333rem in `horizontalLayout` is not the same as `[45.333fr, 25.333fr]` in the media query. 

For better readability could the media query's columns be changed to `[9fr, 5fr]`? I tried that out locally, and think it achieves the same ratio you intended.

Reply:
Sure, but that’s not quite the same as the 45.333rem/25.333rem that the classifier design uses for subject/task widths.

Reply:
.333rem is about 5px at a base font size of 16px, so that might be significant. I’m not sure.